appinventor/docs/reference/components/userinterface.html

```diff
@@ -936,6 +936,8 @@ <h2 id="Screen">Screen</h2>
   <dd>The requested screen orientation. Commonly used values are unspecified (-1), landscape (0), portrait (1), sensor (4), and user (2).  See the Android developer docuemntation for ActivityInfo.Screen_Orientation for the complete list of possible settings.</dd>
   <dt><code>Scrollable</code></dt>
   <dd>When checked, there will be a vertical scrollbar on the screen, and the height of the application can exceed the physical height of the device. When unchecked, the application height is constrained to the height of the device.</dd>
+  <dt><code><em>ShowStatusBar</em></code></dt>
```

Comment:
Regenerate the comments, so this gets the new description of the property.  Or just change this <dd> manually.